Ira/my-app-video/src/components/layout/header/Header.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {Logo} from '../../shared/logo/Logo';
+import {Button} from "../../shared/button/Button";
+import {FullInfoFilmCard} from "../../fullInfoFilmCard";
+import {IHeaderProps} from "../../../interfaces/IHeaderProps";
```

Comment:
I would recommend you to keep IHeaderProps in the same file with Header. Because when you need to change some props in Header, you will need to go to the separate folder and look up for needed interface and this is not really convenient. And you won't probably use this interface in other files.

Reply:
ok